scrapers/ep_votes/scrapers.py

```diff
@@ -137,9 +137,37 @@ def _extract_data(self) -> MemberInfo:
             first_name=first,
             last_name=last,
             date_of_birth=self._date_of_birth(),
+            facebook=self._facebook(),
+            twitter=self._twitter(),
+            email=self._email(),
             country=self._country(),
         )
 
+    def _facebook(self) -> Optional[str]:
+        tag = self._resource.select_one("a.link_fb")
```

Comment:
Could we limit this to the #presentationmep container? `link_fb` seems like a pretty generic class name which might be used somewhere else on the page in the future, so limiting it to the section of the page that’s explicitly about the MEP could make this more robust.

```suggestion
        tag = self._resource.select_one("#presentationmep a.link_fb")
```